package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why do you need `package.json` (and lock) here? I guess someone installed this dependency by mistake

Reply:
removed